src/BookReaderTemplate/template.js

```diff
@@ -0,0 +1,124 @@
+/**
```

Comment:
Call this file BookReaderTemplate.js to match the class and be consistent with how other files are named when they have "main" class (e.g. `BookNavigator.js`)